src/main/java/mil/dds/anet/config/AnetConfiguration.java

```diff
@@ -146,6 +183,57 @@ public void setDictionary(Map<String, Object> dictionary) {
     this.dictionary = Collections.unmodifiableMap(dictionary);
   }
 
+  public void loadDictionary() throws IOException, IllegalArgumentException {
+    // Read and set anet-dictionary
+    final File file = new File(System.getProperty("user.dir"), getAnetDictionaryName());
```

Comment:
Add comment above this line to suppress this false positive from ShiftLeft scan:
```
// scan:ignore — false positive, we *want* to load the user-provided dictionary file
```